app/styles/ilios-common/variables.scss

```diff
@@ -48,3 +48,16 @@ $base-background-color: $background-grey;
 
 // Link Colors
 $base-link-color: $ilios-blue;
+
+// Toggles
+// @todo identify and remove unused variables [ST 2018/11/26]
```

Comment:
I may just not understand scss, but wern't these originally scoped into `toggle-wide` and now they're global here?